This aids in allocating libclang.so from cindex.py. We do not install
libclang.so without version into /usr/lib to allow installing multiple versions.
Therefor, cindex.py should search for the versioned file instead.

--- a/bindings/python/clang/cindex.py
+++ b/bindings/python/clang/cindex.py
@@ -4244,7 +4244,7 @@ class Config:
         elif name == "Windows":
             file = "libclang.dll"
         else:
-            file = "libclang.so"
+            file = "libclang.so.@@SOVER@@"
 
         if Config.library_path:
             file = Config.library_path + "/" + file
